### ISSUE TITLE
Apply default arguments on module tests

### ISSUE THREAD

**Issue by jeroenmaelbrancke** [openvstorage/openvstorage-health-check#474]
# Problems
running ovs healthcheck ovs doesn't apply the default arguments.
Only the single tests apply the arguments.
```
[SUCCESS] DNS resolving works!
[INFO] Checking if log files their size is not bigger than 500.0 MB: 
[SUCCESS] Logfile /var/log//post-update-openvstorage-sdm.log size is fine!
[SUCCESS] Logfile /var/log//alternatives.log size is fine!
[SUCCESS] Logfile /var/log//apt/term.log size is fine!
[SUCCESS] Logfile /var/log//apt/history.log size is fine!
[SUCCESS] Logfile /var/log//unattended-upgrades/unattended-upgrades-shutdown.log size is fine!
[SUCCESS] Logfile /var/log//bootstrap.log size is fine!
[SUCCESS] Logfile /var/log//ovs/scrubber_data-ny1-03.log size is fine!
[SUCCESS] Logfile /var/log//ovs/support.log size is fine!
[SUCCESS] Logfile /var/log//ovs/scrubber_ss-ny1-02.log size is fine!
[SUCCESS] Logfile /var/log//ovs/ovs.log size is fine!
[SUCCESS] Logfile /var/log//ovs/dal.log size is fine!

```

Same for arakoon

```
[SUCCESS] Arakoon cluster globalbackend04-nsm_19 on node 172.17.16.17 should not be collapsed. The oldest tlx is at least 3 days younger than the youngest tlog (actual age: 15:59:28)
[SUCCESS] Spare space for local collapse is sufficient (n > 4x head.db size)
[SUCCESS] Arakoon cluster globalbackend04-nsm_19 on node 172.17.23.9 should not be collapsed. The oldest tlx is at least 3 days younger than the youngest tlog (actual age: 2 days, 0:40:40)
[SUCCESS] Spare space for local collapse is sufficient (n > 4x head.db size)
[SUCCESS] Arakoon cluster globalbackend04-nsm_19 on node 172.17.23.10 should not be collapsed. The oldest tlx is at least 3 days younger than the youngest tlog (actual age: 1 day, 21:48:08)
[SUCCESS] Spare space for local collapse is sufficient (n > 4x head.db size)
[FAILED] Arakoon cluster globalbackend04-nsm_20 on node 172.17.16.9 should be collapsed. The oldest tlx is currently 3 days, 5:30:23 old
[SUCCESS] Spare space for local collapse is sufficient (n > 4x head.db size)
[SUCCESS] Arakoon cluster globalbackend04-nsm_20 on node 172.17.16.10 should not be co
```

default_arguments config:
```
{
    "alba": {
        "disk-safety-test": {
            "include_errored_as_dead": false, 
            "skip_backend": [], 
            "backend": []
        }, 
        "backend-test": {}, 
        "processes-test": {}, 
        "proxy-test": {}, 
        "nsm-load-test": {}, 
        "proxy-port-test": {}
    }, 
    "ipmi": {
        "ipmi-test": {}
    }, 
    "arakoon": {
        "file-descriptors-test": {
            "fd_limit": 30
        }, 
        "ports-test": {}, 
        "integrity-test": {}, 
        "nodes-test": {
            "max_transactions_behind": 10
        }, 
        "collapse-test": {
            "max_collapse_age": 5, 
            "min_tlx_amount": 10
        }
    }, 
    "volumedriver": {
        "volume-potential-test": {
            "critical_vol_number": 25
        }, 
        "sco-cache-mountpoint-test": {}, 
        "dtl-test": {}, 
        "halted-volumes-test": {}
    }, 
    "ovs": {
        "model-test": {}, 
        "log-files-test": {
            "max_log_size": 1
        }, 
        "recovery-domain-test": {}, 
        "nginx-ports-test": {}, 
        "directories-test": {}, 
        "workers-test": {}, 
        "dns-test": {}, 
        "celery-ports-test": {}, 
        "packages-test": {}, 
        "memcached-ports-test": {}, 
        "zombie-processes-test": {}, 
        "verify-rabbitmq-test": {}, 
        "processes-test": {}, 
        "port-ranges-test": {
            "minimal_port_amount": 20
        }
    }
}
```

**Comment by JeffreyDevloo** [comment]
Fixed by #475 
Packaged in openvstorage-health-check_3.8.0-dev.1527610634.03e2674-1_amd64.deb

**Comment by JeffreyDevloo** [comment]
Released in https://github.com/openvstorage/openvstorage-health-check/releases/tag/3.4.10 and https://github.com/openvstorage/openvstorage-health-check/releases/tag/3.8.0
